tests/test-synthetic-resource-bindings.cpp

```
@@ -0,0 +1,646 @@
+    bufferDesc.usage = BufferUsage::ShaderResource | BufferUsage::UnorderedAccess | BufferUsage::CopySource;
+    bufferDesc.defaultState = ResourceState::UnorderedAccess;
+    bufferDesc.memoryType = MemoryType::DeviceLocal;
+
+    ComPtr<IBuffer> buffer;
+    REQUIRE_CALL(device->createBuffer(bufferDesc, initData, buffer.writeRef()));
+    return buffer;
+}
+
+} // namespace
+
+GPU_TEST_CASE("synthetic-resource-bindings-interface-is-opt-in", ALL)
```

Cppcheck (2.20.0)

[error] There is an unknown macro here somewhere. Configuration is required. If DOCTEST_GLOBAL_NO_WARNINGS is a macro then please configure it.

(unknownMacro)